cmds/df/df.go

```diff
@@ -0,0 +1,153 @@
+//
+// df reports details of mounted filesystems
+//
+// Synopsis
+//  df [-k] [-m]
+//
+// Description
+//  read mount information from /proc/mounts and
+//  statfs syscall and display summary information for all
+//  mount points that have a non-zero block count.
+//  Users can choose to see the diplay in KB or MB.
+//
+// Options
+//  -k: display values in KB (default)
+//  -m: dispaly values in MB
+//
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"syscall"
+)
+
+var (
+	inKB  = flag.Bool("k", false, "Express the values in kilobytes (default)")
+	inMB  = flag.Bool("m", false, "Express the values in megabytes")
+	units uint64
+)
+
+const mountinfoFile = "/proc/mounts"
```

Comment:
somewhat confusing name, as there is also a /proc/mountinfo

Reply:
Agreed; will fix - I'll put it in the next commit.